SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -3632,142 +3664,82 @@ void CEulerSolver::SetMax_Eigenvalue(CGeometry *geometry, CConfig *config) {
 
 }
 
-void CEulerSolver::SetUndivided_Laplacian(CGeometry *geometry, CConfig *config) {
+void CEulerSolver::SetUndivided_Laplacian_And_Centered_Dissipation_Sensor(CGeometry *geometry, CConfig *config) {
```

Comment:
These routines were transformed into point loops (no need for grid coloring) and fused since they are always called together. (The code is actually much simpler especially w.r.t. the `boundary_i, boundary_j` conditionals)

Reply:
:+1: